ansible/roles/newrelic/tasks/newrelic.yml

```diff
@@ -0,0 +1,82 @@
+  no_log: true
+
+- name: Pull nri-bundle charts
+  command: helm chart pull gcr.io/domino-eng-service-artifacts/nri-bundle:{{ nribundle_chart_version }}
+  run_once: yes
+  environment:
+    HELM_EXPERIMENTAL_OCI: 1
+
+- name: Export nri-bundle charts to local
+  command: helm chart export gcr.io/domino-eng-service-artifacts/nri-bundle:{{ nribundle_chart_version }} -d {{ local_chart_path }}
+  run_once: yes
+  environment:
+    HELM_EXPERIMENTAL_OCI: 1
+
+- name: Probe bitnami repo
+  command: helm repo add bitnami https://charts.bitnami.com/bitnami
+  when: not (repos.json | json_query("[?name == \"bitnami\"]"))
+
+- name: Probe Newrelic Namespace
+  command: kubectl get ns monitoring
+  changed_when: false
+  ignore_errors: yes
+  run_once: yes
+  register: rancher_monitoring
+
+- name: Create Newrelic namespace
+  command: kubectl create namespace monitoring
+  run_once: yes
+  when: rancher_monitoring is failed
+
+- name: Install Kube State Metrics
+  command: >
+    helm install kube-state-metrics bitnami/kube-state-metrics
+      --namespace {{ newrelic_namespace }}
+      --set securityContext.enabled=true
+      --set securityContext.runAsUser=65534
+      --set securityContext.fsGroup=65534
+  when: not (charts.json | json_query("[?name == \"kube-state-metrics\"]"))
+
+- name: Install Newrelic Services
+  command: >
+    helm install nri-bundle {{ local_chart_path }}/nri-bundle
+      --namespace {{ newrelic_namespace }}
+      --version {{ nribundle_chart_version }}
+      --set global.licenseKey="{{ newrelic_licensekey }}"
+      --set global.cluster="rancher"
+      --set newrelic-infrastructure.enabled=true
+      --set newrelic-infrastructure.rbac.pspEnabled=true
+      --set newrelic-infrastructure.networkPolicy.create=true
+      --set newrelic-infrastructure.resources.limits.memory=128Mi
+      --set newrelic-infrastructure.resources.requests.cpu=100m
+      --set newrelic-infrastructure.resources.requests.memory=30Mi
+      --set kubeEvents.enabled=true
+      --set newrelic-events.networkPolicy.create=true
+      --set newrelic-events.resources.limits.memory=128Mi
+      --set newrelic-events.resources.requests.cpu=100m
+      --set newrelic-events.resources.requests.memory=128Mi
+      --set logging.enabled=true
+      --set newrelic-logging.rbac.pspEnabled=true
+      --set newrelic-logging.networkPolicy.enabled=true
+      --set newrelic-logging.resources.limits.memory=256Mi
+      --set newrelic-logging.resources.requests.cpu=100m
+      --set newrelic-logging.resources.requests.memory=30Mi
+      --set prometheus.enabled=true
+      --set newrelic-open-metrics.resources.limits.cpu=1
+      --set newrelic-open-metrics.resources.limits.memory=1Gi
+      --set newrelic-open-metrics.resources.requests.cpu=100m
+      --set newrelic-open-metrics.resources.requests.memory=30Mi
+      --description='Deploy Newrelic services'
+      --set newrelic-infrastructure.customAttributes="'{\"clusterName\":\"$(CLUSTER_NAME)\"}'"
```

Comment:
Can we set something like `role=rancher` as well?

Reply:
We are setting the `clusterName` as `rancher`. Do we still need to set the role too?